### ISSUE TITLE
Configurable auto-correct format for @param

### ISSUE THREAD

**Issue by renatolond** [ksss/rubocop-yard#14]
I noticed that when using the yard auto correct, it auto-corrects with rdoc formatting:
``` 
# @param param1 [Object] My nice param
def my_method(param1, param2)
end
```
After `rubocop -a`:
```
# @param param1 [Object] My nice param
# @param [Object] param2
def my_method(param1, param2); end
```

Could the formatting (rdoc/yard) for params be configurable through `rubocop.yml`?

**Comment by ksss** [comment]
As you said, it seems that with the `@param` tag, the name can be either before or after the type.

https://rubydoc.info/gems/yard/file/docs/Tags.md
> freeform data beginning with an optional types list, as well as a name key, placed either before or after the types list.

I followed this style because many of the method documents within the lsegal/yard repository had the name written after the type.


https://rubydoc.info/gems/yard/file/README.md
> YARD uses a '@tag' style definition syntax for meta tags alongside regular code documentation. These tags should be able to happily sit side by side RDoc formatted documentation, but provide a much more consistent and usable way to describe important information about objects, such as what parameters they take and what types they are expected to be, what type a method should return, what exceptions it can raise, if it is deprecated, etc.. It also allows information to be better (and more consistently) organized during the output generation phase.

Indeed, since both formats are valid, I agree that it would be beneficial to have the option to choose between them.
However, I couldn't find any reference stating that one style is referred to as the 'rdoc' format.
While it's true that YARD is partially compatible with rdoc, the tags are described as being specific to YARD, so I cannot agree to name the formats (rdoc/yard).

How about naming the options (before/after) instead?

**Comment by renatolond** [comment]
Ah, yeah, you're right! I saw that about `@param` in a rdoc cheat sheet some time and assumed it came from there. before/after makes perfect sense to me 👍🏼

**Comment by ksss** [comment]
@renatolond 
I released [v0.8.0](https://rubygems.org/gems/rubocop-yard/versions/0.8.0)
Please check it out!

**Comment by renatolond** [comment]
Works perfectly, thanks! ❤️
